CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ set(UR_ADAPTER_HIP_SOURCE_DIR "" CACHE PATH
     "Path to external 'hip' adapter source dir")
 set(UR_ADAPTER_NATIVE_CPU_SOURCE_DIR "" CACHE PATH
     "Path to external 'native_cpu' adapter source dir")
+option(UR_BUILD_XPTI_LIBS "Build the XPTI libraries when tracing is enabled" ON)
```

Comment:
NIT: We could move this line to line 50 to be near other options.

Reply:
I think we could make it by default have the same value as `UR_ENABLE_TRACING` as so if tracing got enabled, this would be enabled too. and we could also control it manually be assigning it explicitly if we need.

Reply:
I've moved the option but kept it as defaulted to `ON`. If it defaults to the value of `UR_ENABLE_TRACING` and a user configures with tracing off, but then later enables tracing, then the option would still be `OFF` which would be surprising if they're not aware of this particular option.

Reply:
Make sense.